src/pages/iou/request/step/IOURequestStepDate.js

```diff
@@ -67,7 +67,8 @@ function IOURequestStepDate({
     const {translate} = useLocalize();
 
     // In the split flow, when editing we use SPLIT_TRANSACTION_DRAFT to save draft value
```

Comment:
@brunovjk Move this comment to before isEditingSplitBill definition

Reply:
Sorry about that. Done :D